pkg/controller/controller_utils.go

```diff
@@ -51,6 +51,7 @@ import (
 	"k8s.io/utils/integer"
 
 	"k8s.io/klog"
+	"strconv"
```

Comment:
Wrong order of headers.

Reply:
This part comes from the original commit https://github.com/kubernetes/kubernetes/pull/75763/commits